docs/OV_Runtime_UG/Python_API_exclusives.md

```diff
@@ -0,0 +1,133 @@
+# OpenVINO™ Python API exclusives {#openvino_docs_OV_Runtime_UG_Python_API_exclusives}
```

Comment:
this page is not connected with other documentation. Need to put the `openvino_docs_OV_Runtime_UG_Python_API_exclusives` reference to any other document to make a connection